pages/api/availability/[user].ts

```diff
@@ -34,6 +35,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
             requestBody: {
                 timeMin: req.query.date + "T00:00:00.00Z",
                 timeMax: req.query.date + "T23:59:59.59Z",
+                timeZone: currentUser.timeZone,
```

Comment:
This _may_ not work due to the UTC formatting in timeMin and timeMax, but I need to confirm this. Will come back to this later.

Reply:
I think you are probably right.

Google Calendar API Docs says `timeZone | string | Time zone used in the response. Optional. The default is UTC.` so it is just the return format that is changing, but `timeMin` and `timeMax` are still in UTC.